srcpkgs/cherrytree/template

```diff
@@ -13,7 +13,10 @@ maintainer="Logen K <logen@sudotask.com>"
 license="GPL-3.0-or-later"
 homepage="https://www.giuspen.com/cherrytree/"
 distfiles="https://github.com/giuspen/cherrytree/archive/$version.tar.gz"
-checksum=5c180fdf5cbab43bc496a246ce952e7f39e84b2a7b4cabdb22773b9d4a415850
+checksum=fff13c6b764eaa952616308a53cb5bd863b37f913bd74891117b4da20ff29832
+configure_args+=" -DBUILD_GMOCK:BOOL='OFF' \
+		-DBUILD_GTEST:BOOL='OFF' \
+		-DBUILD_TESTING:BOOL='OFF'"
```

Comment:
```suggestion
configure_args+=" -DBUILD_GMOCK:BOOL='OFF'
 -DBUILD_GTEST:BOOL='OFF' 
 -DBUILD_TESTING:BOOL='OFF'"
```

Well, tests broke because you stopped building them :P 

Put this block inside a conditional like `[ -z "$XBPS_CHECK_PKGS" ]`. This is worth it only if the tests take long to build; otherwise it's easier to just allow them to build.

Reply:
The problem I ran into is that it pulls in gtest as a submodule (See: https://github.com/giuspen/cherrytree/tree/master/tests). It's not included in the source tarball, so I wasn't sure how to go about it.